src/commitlint/index.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  Rule,
+  SchematicContext,
+  Tree,
+  chain,
+  mergeWith,
+  apply,
+  url,
+  move
+} from '@angular-devkit/schematics';
+import { PackageJson, installDependencies } from '../lib';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+
+export default function commitlint(_options: any): Rule {
+  return chain([
+    installDependencies({
+      devDependencies: [
```

Comment:
I really like this `Rule`.

Nevertheless, it should be able to set the version as well.
But we do not have to change it right now.

Reply:
Can add it to the feature you want to create for the [latest version problem](https://github.com/co-IT/schematics/pull/5#discussion_r)